.env

```diff
@@ -13,5 +15,11 @@ LISTENER_ENV=development
 WES_VERSION=3.3
 WES_PORT=5000
 WES_BACKEND=wes_service.cwl_runner
-WES_CWL_RUNNER=cwltoil
+WES_CWL_RUNNER=cwltool
 WES_LOG=DEBUG
+
+DOS_PORT=8080
```

Comment:
For the purposes of the demo, we can keep the `DOS_*` variables but eventually these will need to be removed once ga4gh_dos is deprecated

Reply:
Should I add a 'boolean' env var that states whether to use ga4gh dos or chord drs and generalize the listener to use either depending on that 'boolean' env var?

Reply:
Separate the env variables for GA4GH_DOS and CHORD_DRS

Reply:
Solved in e3b0539 by switching entirely to chord's drs while leaving most of the components of ga4gh's dos